README.md

```diff
@@ -283,6 +186,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
Currently, the link is still broken and shows a 404 ERROR on clicking the link this is because the name of the file is misspelled to [LINCENSE](https://github.com/noelfoka/Hello-Microverse/blob/hello-microverse/LINCENSE). Now you can either rename the file to [LICENSE]() or make a change in the link file path. Please consider resolving it as it is a requirement to complete the project.